client/pages/layout.jsx

```diff
@@ -1,24 +1,34 @@
 import React from 'react'
 import { Provider } from 'react-redux'
-import { Navbar } from '../components/navbar'
+import Navbar from '../components/navbar'
 import PageTransition from '../components/page-transition'
 import Footer from '../components/footer'
 import UniversalLoader from '../components/universal-loader'
 import store from '../redux/store'
+import { setPreferredSlideDirection } from '../redux/action-creators'
 
 let pageBack = function () {
-  let current = FlowRouter.current()
-  let parent = current.route.options.parent
-  if (!parent) return
-  FlowRouter.go(parent, current.params, current.queryParams)
+  let details = store.getState().pageDetails
+  if (details.parent && details.parent.name) {
+    store.dispatch(setPreferredSlideDirection('right'))
+    FlowRouter.go(details.parent.name, details.parent.params, details.parent.queryParams)
+  }
+}
+
+let pageForward = function () {
+  let details = store.getState().pageDetails
+  if (details.child && details.child.name) {
+    store.dispatch(setPreferredSlideDirection('left'))
+    FlowRouter.go(details.child.name, details.child.params, details.child.queryParams)
+  }
 }
```

Comment:
Although this is a stateless functional component, which is nice, so I'll probably just put it into PageTransition...